docs/migration_notes.md

```diff
@@ -6,6 +6,16 @@ keywords: [Noir, notes, migration, updating, upgrading]
 
 Noir is in full-speed development. Things break fast, wild, and often. This page attempts to leave some notes on errors you might encounter when upgrading and how to resolve them until proper patches are built.
 
+## ≥0.14
+
+The index of the [for loops](./language_concepts/02_control_flow.md#loops) is now of type `u64` instead of `Field`. An example refactor would be:
```

Comment:
Can we silence the warnings from the compiler about this yet?

Reply:
Just checked, and yes the compiler stops warning about it from 0.14.0 onwards